src/io/keyword/easyevents/Sessions.java

```diff
@@ -0,0 +1,51 @@
+package io.keyword.easyevents;
+
+import java.time.LocalTime;
+
+/**
+ * Class Description
+ * <p>
+ * Created By: Justin Reeves, Pasang Sherpa, Xiaotian Wang (Xander)
+ * Created On: 3/26/2021
+ */
+class Sessions {
+
+    // FIELDS
+    private LocalTime initialTime;
+    private EventLog eventslog = EventLog.getInstance();
+    private String sessionName;
+    private static final Sessions instance = new Sessions();
+
+
+    // CONSTRUCTORS
+
+
+    private Sessions() {
+    }
+
```

Comment:
Need ctor for (String name), (LocalTime initialTime), (String name, LocalTime intialTime)